protocol/x/clob/abci.go

```diff
@@ -89,30 +89,10 @@ func EndBlocker(
 		)
 	}
 
-	deliveredCancels := keeper.GetDeliveredCancelledOrderIds(ctx)
-	// Prune expired untriggered conditional orders from the in-memory UntriggeredConditionalOrders struct.
-	keeper.PruneUntriggeredConditionalOrders(
-		expiredStatefulOrderIds,
-		deliveredCancels,
-	)
-
 	// Update the memstore with expired order ids.
 	// These expired stateful order ids will be purged from the memclob in `Commit`.
 	processProposerMatchesEvents.ExpiredStatefulOrderIds = expiredStatefulOrderIds
 
-	// Before triggering conditional orders, add newly-placed conditional orders to the clob keeper's
-	// in-memory UntriggeredConditionalOrders data structure to allow conditional orders to
-	// trigger in the same block they are placed. Skip triggering orders which have been cancelled
-	// or expired.
-	// TODO(CLOB-773) Support conditional order replacements. Ensure replacements are de-duplicated.
-	conditionalOrdersIds := keeper.GetDeliveredConditionalOrderIds(ctx)
```

Comment:
Don't think `GetDeliveredConditionalOrderIds` is used for anything else? Should we remove it as well?